docs/examples/DataSet/Dataset Context Manager.ipynb

```diff
@@ -131,10 +140,10 @@
     "\n",
```

Comment:
Should we link the example notebook for the `The Experiment Container` here? http://qcodes.github.io/Qcodes/examples/DataSet/The-Experiment-Container.html